thingsboard_gateway/storage/sqlite/database.py

```diff
@@ -95,6 +104,8 @@ def process(self):
 
                         self.db.commit()
 
+                    sleep(.2)
```

Comment:
Probably we have to sleep when we don't process any data instead of sleeping after every processing, what do you think?